pkg/sensors/base/base.go

```diff
@@ -92,6 +105,10 @@ func setupPrograms() {
 		}
 	}
 	logger.GetLogger().Infof("Exit probe on %s", Exit.Attach)
+
+	threads := readFileDefault("/proc/sys/kernel/threads-max", 32768)
+	ExecveMap.SetMaxEntries(int(threads))
```

Comment:
Why do we need this? It seems it will make the size of the execve_map even larger as threads-max is often well above 32K. This will take significant space while running threads-max threads is pretty rare nop?

Reply:
I tried this as mitigation for https://github.com/isovalent/security/issues/88  .. I think we need some combination of this change (with some reasonable size for execve_map) and other ways mentioned in the issue

Reply:
ah I see, maybe this is one of the cases where NO_PREALLOC could help so that we can dynamically size to a very large map. But I could see how this can lead to memory issues in the future. That's not an easy problem :/